docs/src/components/landing-page/TwitterShowcaseCards.vue

```diff
@@ -36,65 +37,82 @@
     </template>
     <q-carousel-slide :name="slideIndex" class="showcase-cards text-size-10" v-for="(tweetGroup, slideIndex) in tweetGroups" :key="`slide-${slideIndex}`">
       <div class="carousel-grid">
-        <div v-for="(tweetId, cardIndex) in tweetGroup" :key="`twitter-card-${cardIndex}`">
-          <twitter-card :tweetId="tweetId"/>
+        <div v-for="(tweetId, cardIndex) in tweetGroup" :key="`twitter-card-${cardIndex}`" class="tweeter-tweet">
+          <div :id="`tweet-container-${tweetId}`"></div>
         </div>
       </div>
     </q-carousel-slide>
   </q-carousel>
 </template>
 
-<script>
-import { defineComponent, ref } from 'vue'
-import TwitterCard from 'src/components/landing-page/TwitterCard'
+<script lang="ts">
+import { defineComponent, onMounted, ref } from 'vue'
 
-// TODO: fix the issue causing ONLY the first tweet group (slide 1) display
 const scriptElement = document.createElement('script')
 scriptElement.setAttribute('src', 'https://platform.twitter.com/widgets.js')
 scriptElement.setAttribute('charset', 'utf-8')
 document.head.appendChild(scriptElement)
 
+const addTwttr = async () => {
+  return new Promise(resolve => {
+    scriptElement.onload = () => {
+      resolve(window.twttr)
+    }
+  })
+}
+
 export default defineComponent({
   name: 'TwitterShowcaseCards',
-  components: { TwitterCard },
   setup () {
     const slide = ref(0)
+    let twttr = null
+    const groupIsDisplayed = []
+
     const tweetGroups = [
       [
         '1453670879825629189',
-        '1453670879825629189',
-        '1453670879825629189'
-      ],
-      [
-        '1453670879825629189',
-        '1453670879825629189',
-        '1453670879825629189'
-      ],
-      [
-        '1453670879825629189',
-        '1453670879825629189',
-        '1453670879825629189'
-      ],
-      [
-        '1453670879825629189',
-        '1453670879825629189',
-        '1453670879825629189'
-      ],
-      [
-        '1453670879825629189',
-        '1453670879825629189',
-        '1453670879825629189'
+        '1429853761422364681'
       ],
       [
-        '1453670879825629189',
-        '1453670879825629189',
-        '1453670879825629189'
+        '1419939610067623937'
       ]
     ]
 
+    const createTweetCard = (twttr, tweetId, tweetContainerId) => {
+      twttr.widgets.createTweet(
+        tweetId,
+        document.getElementById(tweetContainerId),
+        {
+          theme: 'light',
+          conversation: 'none',
+          cards: 'hidden'
+        }
+      )
+    }
+
+    onMounted(async () => {
+      twttr = await addTwttr()
+      // display first tweet group
+      tweetGroups[ 0 ].forEach(tweetId => {
+        createTweetCard(twttr, tweetId, `tweet-container-${tweetId}`)
+      })
+      groupIsDisplayed.push(0)
+    })
+
+    const loadTweets = () => {
```

Comment:
Prefer using `function` over declaring a `const`, unless the given function is really simple and directly returns a value (eg. wrapper fuctions, mapping functions, etc)
Here you don't spare a single char, so just use `function` which is what most people would expect to be used